src/views/StoriesList.vue

```diff
@@ -145,43 +91,64 @@ function viewStory(storyId) {
   <v-container>
     <div id="body">
       <v-row align="center" class="mb-4">
-        <v-col cols="10"><v-card-title class="pl-0 text-h4 font-weight-bold">Stories</v-card-title>
+        <v-col cols="10">
+          <v-card-title class="pl-0 text-h4 font-weight-bold">Stories</v-card-title>
         </v-col>
         <v-col class="d-flex justify-end" cols="2">
-          <v-btn v-if="user !== null" color="accent" @click="openAdd()">Add</v-btn>
+          <v-btn v-if="user !== null" color="accent" @click="openAdd">Add</v-btn>
         </v-col>
       </v-row>
 
-      <StoryCard v-for="story in stories" :story="story" @storySelected="viewStory" />
+      <StoryCard v-for="story in stories" :story="story" @click="viewStory(story.storyId)"/>
 
       <v-dialog persistent v-model="isAdd" width="800">
         <v-card class="rounded-lg elevation-5">
-          <v-card-title class="headline mb-2">Add Recipe </v-card-title>
+          <v-card-title class="headline mb-2">Add Story</v-card-title>
           <v-card-text>
-            <v-text-field v-model="newRecipe.name" label="Name" required>
-            </v-text-field>
-
-            <v-text-field v-model.number="newRecipe.servings" label="Number of Servings" type="number">
-            </v-text-field>
-            <v-text-field v-model.number="newRecipe.time" label="Time to Make (in minutes)" type="number">
-            </v-text-field>
-
-            <v-textarea v-model="newRecipe.description" label="Description">
-            </v-textarea>
-            <v-switch v-model="newRecipe.isPublished" hide-details inset
-              :label="`Publish? ${newRecipe.isPublished ? 'Yes' : 'No'}`"></v-switch>
+            <v-text-field
+              v-model="newStory.setting"
+              label="Story Setting"
+              required
+            ></v-text-field>
```

Comment:
these lines which begin with the closing greater than sign just really stand out and look odd to me